certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/fk7443/CertificateModelFactoryFK7443.java

```diff
@@ -8,6 +8,7 @@
 import static se.inera.intyg.certificateservice.domain.certificatemodel.model.WorkCapacityType.TRE_FJARDEDELAR;
 
 import java.time.LocalDateTime;
+import java.time.Period;
```

Comment:
For date validation we had min set as type TemporalAmount. Do we want to have the same here?